### PR TITLE
ORC: Add ORC bloom filter attributes to iceberg table

### PR DESCRIPTION
**Pull request by zhengxingmao**
Hi ALL，this PR add ORC bloom filter attributes to iceberg table.In this way, the upper computing engine (Presto/Trino) can implement the orc bloom filter function

### CODE REVIEW

File: core/src/main/java/org/apache/iceberg/TableProperties.java [hunk 1]
```diff
@@ -129,6 +129,8 @@ private TableProperties() {
   public static final String PARQUET_BATCH_SIZE = "read.parquet.vectorization.batch-size";
   public static final int PARQUET_BATCH_SIZE_DEFAULT = 5000;
 
+  public static final String ORC_BLOOM_FILTER_COLUMNS = "orc_bloom_filter_columns";
```
Comment (hililiwei) [inline core/src/main/java/org/apache/iceberg/TableProperties.java:132]: "orc.bloom.filter.columns"

File: core/src/main/java/org/apache/iceberg/TableProperties.java [hunk 2]
```diff
@@ -129,6 +129,8 @@ private TableProperties() {
...
+  public static final String ORC_BLOOM_FILTER_FPP = "orc_bloom_filter_fpp";
```
Comment (hililiwei) [inline core/src/main/java/org/apache/iceberg/TableProperties.java:133]: "orc.bloom.filter.fpp"

**Review by hililiwei — CHANGES_REQUESTED** [review]
Thanks for your contributions, I left a few comments. Also, could you update the document at the same time?

**Comment by jackye1995** [comment]
how would this be used by upstream? Could you describe in more details?

**Comment by zhengxingmao** [comment]
> how would this be used by upstream? Could you describe in more details?

Take the iceberg connector of Presto / Trino as an example, if Orc is not added ORC_ BLOOM_ FILTER_ Columns and orc_ BLOOM_ FILTER_ FPP table attribute, Presto / Trino cannot specify bloom filter column when creating iceberg table.

File: core/src/main/java/org/apache/iceberg/TableProperties.java (hunk 2 above)
Comment (zhengxingmao) [reply core/src/main/java/org/apache/iceberg/TableProperties.java]: Thank you for your review. It has been corrected according to the suggestions and the docs description has been added.

File: core/src/main/java/org/apache/iceberg/TableProperties.java [hunk 3]
```diff
@@ -129,6 +129,8 @@ private TableProperties() {
   public static final String PARQUET_BATCH_SIZE = "read.parquet.vectorization.batch-size";
   public static final int PARQUET_BATCH_SIZE_DEFAULT = 5000;
 
+  public static final String ORC_BLOOM_FILTER_COLUMNS = "orc.bloom.filter.columns";
+  public static final String ORC_BLOOM_FILTER_FPP = "orc.bloom.filter.fpp";
```
Comment (rdblue) [inline core/src/main/java/org/apache/iceberg/TableProperties.java:133]: If there's nothing that implements these properties, then I don't think that we need to include them.

**Comment by rdblue** [comment]
I'm going to close this since there's no implementation associated with these properties, and they aren't Iceberg config properties. If you want, you could document that Hadoop properties set in the table's properties are passed through so setting things like this will work.

File: core/src/main/java/org/apache/iceberg/TableProperties.java (hunk 3 above)
Comment (zhengxingmao) [reply core/src/main/java/org/apache/iceberg/TableProperties.java:133]: These two parameters are used for Trino, not iceberg. The relevant implementation code is in Trino.

**Pull request closed without merging**